src/main/java/com/sh/threesentences/auth/service/AuthService.java

```diff
@@ -0,0 +1,28 @@
+package com.sh.threesentences.auth.service;
+
+
+import com.sh.threesentences.auth.dto.LoginDto;
+import javax.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+@Service
+@Transactional
```

Comment:
```suggestion
@Transactional(readOnly = true)
```

이후에 쓰기 작업이 추가되면 문제가 될 순 있지만 AuthService의 특성상 그러지 않을 것 같아서 추가하면 좋을 것 같네요.